### PR TITLE
More complex instantiations

### PR DESCRIPTION
**Pull request by bangerth**
In this case for `VectorTools::interpolate_boundary_values`.

Part of #2033.

### CODE REVIEW

**Comment by bangerth** [comment]
I took the liberty to also move one test from `numerics/` to `vector_tools/` where it fits better.

/run-tests

File: tests/vector_tools/boundaries_complex.cc [hunk 1]
```diff
@@ -0,0 +1,183 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2000 - 2015 by the deal.II authors
```
Comment (masterleinad) [inline tests/vector_tools/boundaries_complex.cc]: Do you want to update this?

File: tests/vector_tools/boundaries_complex.cc [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+
+/* Author: Wolfgang Bangerth, University of Heidelberg, 2001 */
```
Comment (masterleinad) [inline tests/vector_tools/boundaries_complex.cc]: Do you want to update this?

File: tests/vector_tools/boundaries.cc [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+
+/* Author: Wolfgang Bangerth, University of Heidelberg, 2001 */
```
Comment (masterleinad) [inline tests/vector_tools/boundaries.cc]: Do you want to update this?

File: tests/vector_tools/boundaries.cc [hunk 4]
```diff
@@ -0,0 +1,180 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2000 - 2015 by the deal.II authors
```
Comment (masterleinad) [inline tests/vector_tools/boundaries.cc]: Do you want to update this?

Comment (bangerth) [reply tests/vector_tools/boundaries.cc]: That's not wrong -- I copied the test from elsewhere, so the majority of the code indeed has these years.

**Comment by bangerth** [comment]
Let me take care of the author notes on a separate branch (I already have follow-up commits on the branch in question here). Patch coming in a second.

**Pull request merged**